README.md

```diff
@@ -69,4 +84,15 @@ Build using Docker
 
 ```
 docker run -it -v $PWD:/scylla-machine-image -w /scylla-machine-image  --rm centos:7.2.1511 bash -c './dist/redhat/build_rpm.sh -t centos7 -c aws'
-```
\ No newline at end of file
+```
+
+## Building docs
+
+```bash
+python3 -m .venv
+source .venv/bin/activate
+pip install sphinx sphinx-jsondomain sphinx-markdown-builder
```

Comment:
I'm getting the following
`ERROR: sphinx-jsondomain 0.0.3 has requirement Sphinx<2,>=1.4, but you'll have sphinx 3.0.2 which is incompatible.`
but the HTML still created correctly AFAICS